src/interaction/navigationhandler.cpp

```diff
@@ -142,7 +150,24 @@ NavigationHandler::NavigationState::NavigationState(std::string anchor, std::str
 NavigationHandler::NavigationHandler()
     : properties::PropertyOwner({ "NavigationHandler" })
     , _useKeyFrameInteraction(KeyFrameInfo, false)
+    , _origin(OriginInfo)
 {
+    _origin.onChange([this]() {
```

Comment:
This should probably be removed as well. And the property itself in the header file.

Reply:
Fixed now